localization/ndt_scan_matcher/README.md

```diff
@@ -59,3 +64,90 @@ There are two main functions in this package:
 | `converged_param_transform_probability` | double | Threshold for deciding whether to trust the estimation result                                   |
 | `omp_neighborhood_search_method`        | int    | neighborhood search method in OMP (0=KDTREE, 1=DIRECT26, 2=DIRECT7, 3=DIRECT1)                  |
 | `omp_num_threads`                       | int    | Number of threads used for parallel computing                                                   |
+
+## Regularization
+
+### Abstract
+
+This is a function that adds the regularization term to the NDT optimization problem as follows.
+
+$$
+\begin{align}
+    \min_{\mathbf{R},\mathbf{t}}
+    \mathrm{NDT}(\mathbf{R},\mathbf{t})
+    +\mathrm{scale\ factor}\cdot \left\|
+        \begin{pmatrix}
+            1&0&0\\
+            0&1&0\\
+            0&0&0
+        \end{pmatrix}
+        (\mathbf{t}-\mathbf{t_{base}})
+        \right\|_{2}^2
+\end{align}
+$$
+
+, where t_base is base position measured by GNSS or other means.
+NDT(R,t) stands for the pure NDT cost function.
+The regularization term shifts the optimal solution to the base position.
```

Comment:
@KYabuuchi 
Thank you for adding to the README.
I think it's almost correct, but I don't think there is enough explanation for this part.
The regularization term shifts the optimal solution towards the base position only in the longitudinal axis of vehicle.
[ndt_omp source code](https://github.com/tier4/ndt_omp/blob/35ea99ad594cbc72233d6269132db1cf7d45bf49/include/pclomp/ndt_omp_impl.hpp#L342)

Reply:
Thanks! I was confused. I will fix it later :runner:

Reply:
@hiro-ya-iv I have corrected the issues you pointed out.
Please check again in your free time :bow:

Reply:
![LGTM](https://image.lgtmoon.dev/168694)